include/execution.hpp

```diff
@@ -3441,17 +3441,17 @@ namespace _P2300::execution {
 
         run_loop* __loop_;
       };
-      __scheduler get_scheduler() {
+      __scheduler get_scheduler() noexcept {
         return __scheduler{this};
       }
 
-      void run();
+      void run() noexcept;
 
-      void finish();
+      void finish() noexcept;
 
      private:
-      void __push_back_(__impl::__task* __task);
-      __impl::__task* __pop_front_();
+      void __push_back_(__impl::__task* __task) noexcept;
+      __impl::__task* __pop_front_() noexcept;
```

Comment:
These are not `noexcept` intentionally. Locking a mutex can throw an exception. The mind boggles at the implication of that, but there it is.